### DISCUSSION TITLE
docs/web/response-buffering

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#82] [General]
# docs/web/response-buffering

在GoFrame框架中如何实现数据返回的缓冲控制。通过使用缓冲区可以提高执行效率并提供更灵活的输出控制。示例代码展示了如何通过中间件统一处理返回数据，避免错误信息直接暴露给客户端，并提供自定义的错误信息提示。

https://goframe.org/docs/web/response-buffering

**Comment by wangle201210** [comment]
> 其中 Output 作用类似于 Flush 将缓冲区的数据输出到客户端并清空缓冲区。
这句话的描述有问题，方法里面没有Output
